overlord/ifacestate/helpers.go

```diff
@@ -88,12 +88,24 @@ func (m *InterfaceManager) addBackends(extra []interfaces.SecurityBackend) error
 }
 
 func (m *InterfaceManager) addSnaps(snaps []*snap.Info) error {
+	var repoSnaps int
```

Comment:
I would find

    repoSnaps := 0

easier to read. But maybe your version is more idiomatic?